addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml

```diff
@@ -4,17 +4,17 @@
     <t t-name="mail.ActivityMarkDonePopover" owl="1">
         <div class="o_ActivityMarkDonePopover" t-attf-class="{{ className }}" t-on-keydown="activityMarkDonePopoverView and activityMarkDonePopoverView.onKeydown" t-ref="root">
             <t t-if="activityMarkDonePopoverView">
-                <textarea class="form-control o_ActivityMarkDonePopover_feedback" rows="3" placeholder="Write Feedback" t-on-blur="activityMarkDonePopoverView.onBlur" t-ref="feedbackTextarea"/>
-                <div class="o_ActivityMarkDonePopover_buttons">
+                <textarea class="form-control o_ActivityMarkDonePopover_feedback mb-3" rows="3" placeholder="Write Feedback" t-on-blur="activityMarkDonePopoverView.onBlur" t-ref="feedbackTextarea"/>
+                <div class="o_ActivityMarkDonePopover_buttons mb-2">
                     <button type="button" class="o_ActivityMarkDonePopover_doneScheduleNextButton btn btn-sm btn-primary" t-on-click="activityMarkDonePopoverView.onClickDoneAndScheduleNext">
                         Done &amp; Schedule Next
                     </button>
                     <t t-if="activityMarkDonePopoverView.activityViewOwner.activity.chaining_type === 'suggest'">
-                        <button type="button" class="o_ActivityMarkDonePopover_doneButton btn btn-sm btn-primary" t-on-click="activityMarkDonePopoverView.onClickDone">
+                        <button type="button" class="o_ActivityMarkDonePopover_doneButton btn btn-sm btn-primary mx-2" t-on-click="activityMarkDonePopoverView.onClickDone">
                             Done
                         </button>
                     </t>
-                    <button type="button" class="o_ActivityMarkDonePopover_discardButton btn btn-sm btn-link" t-on-click="activityMarkDonePopoverView.onClickDiscard">
+                    <button type="button" class="o_ActivityMarkDonePopover_discardButton btn btn-sm btn-link pr-0" t-on-click="activityMarkDonePopoverView.onClickDiscard">
```

Comment:
Yes,
> so if the goal is to "align" the text on the right, it will not even be reliable

indeed, moreover this change was intrinsically wrong, it's not the right way to handle these scenarios.
Thanks